components/ILIAS/Forum/PRIVACY.md

```diff
@@ -0,0 +1,101 @@
+# Forum Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](../../../docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+The forum is an ILIAS repository object in which users can hold discussions. A forum is organized in different topics (threads), to which contributions (posts) are written. As long as a forum is not deleted, posts can be viewed for years after they have been published.
```

Comment:
Can you slightly change to:
"The forum is an ILIAS repository object where users can hold discussions. A forum is organized into different topics (threads), to which contributions (posts) are written."